src/config/Config.zig

```diff
@@ -5882,6 +5898,35 @@ pub const ShellIntegrationFeatures = packed struct {
     title: bool = true,
 };
 
+/// SSH integration levels for shell integration.
+/// Controls how much SSH integration is performed when connecting to remote hosts.
+///
+/// Allowable values are:
+///
+///   * `off` - No SSH integration, use standard ssh command
+///
+///   * `term-only` - Only fix TERM compatibility (xterm-ghostty -> xterm-256color)
+///
+///   * `basic` - TERM fix + environment variable propagation
+///
+///   * `full` - All features: TERM fix + env vars + terminfo installation + shell integration injection
+///
+/// The default value is `off`.
```

Comment:
This isn't necessary as currently these types are not linked to the config generation in any way.

```suggestion
/// See ssh-integration
```

Reply:
Fixed in https://github.com/ghostty-org/ghostty/pull/7608/commits/b9234bc3866f3e2a315bb5f83328d7f8298fcd07